sdk/src/main/java/io/opentelemetry/sdk/common/ConfigBuilder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.common;
+
+import java.util.Map;
+
+public abstract class ConfigBuilder {
```

Comment:
```suggestion
public abstract class ConfigurableBuilder {
```

Looking at the usages, this is not a builder for configuration, but the interface for a configurable builder. Consequently, I think the getProperty methods must be removed from here (they can be moved to a static helper class) and then this can become an interface instead of an abstract class.

Reply:
Agree 100% with @Oberon00 's approach. We should try not to introduce class hierarchies when we can avoid it.